### DISCUSSION TITLE
Welcome to Acer-Nitro-5-AN515-54-Monterey Discussions!

### DISCUSSION THREAD

**Discussion by itsmeshibintmz** [itsmeshibintmz/Acer-Nitro-5-AN515-54-Monterey#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vaibhavkuks** [comment]
Heyyy! Can you please tell how to fix this?
![Screen Shot 2022-05-29 at 3 43 50 PM](https://user-images.githubusercontent.com/67002144/170862928-ffd80708-45c9-41f4-b559-658c10c898f9.png)

Even after signing in here.. it still gets stuck on this step
![Screen Shot 2022-05-29 at 3 44 40 PM](https://user-images.githubusercontent.com/67002144/170862962-d90f55b1-3644-4222-932e-fb29e40ea807.png)

I am not able to update any apps through appstore...

**Reply by itsmeshibintmz** [reply]
https://support.apple.com/en-us/HT204030

Do checkout!

**Comment by vaibhavkuks** [comment]
Nope didn’t help. I’ve tried that. I can download apps but not update them from appstore... which is wierd 😺. That’s why I asked.

**Comment by vaibhavkuks** [comment]
Are you able to normally update apps from the appstore?

**Reply by itsmeshibintmz** [reply]
> Are you able to normally update apps from the appstore?

yes.

**Comment by itsmeshibintmz** [comment]
Have you tried using a different model(SMBIOS) as I suggested?

**Reply by itsmeshibintmz** [reply]
> Have you tried using a different model(SMBIOS) as I suggested?

If you've tried this already, try RESET NVRAM. 

Also if that doesn't work, I suggest a clean install again with the SMBIOS I given in the EFI.(MacBook Pro 16,1).

**Comment by vaibhavkuks** [comment]
I'll try the steps and update you later 👍

**Reply by itsmeshibintmz** [reply]
Good luck.

**Comment by itsmeshibintmz** [comment]
@VaibhavKukreti-Kathud any updates? Is your issue fixed?
